tokio/src/net/driver/reactor/mod.rs

```diff
@@ -81,7 +68,7 @@ thread_local! {
     static CURRENT_REACTOR: RefCell<Option<Handle>> = RefCell::new(None)
 }
 
-const TOKEN_WAKEUP: mio::Token = mio::Token(MAX_SOURCES);
+const TOKEN_WAKEUP: mio::Token = mio::Token(Address::NULL);
```

Comment:
I believe `MAX_SOURCES` is a valid token.  It also occurs to me that I defined `Address::NULL` as `usize::MAX` which won't work as that conflicts w/ an internal token...